targetd.te

```diff
@@ -56,6 +57,8 @@ kernel_request_load_module(targetd_t)
 kernel_dgram_send(targetd_t)
 
 rpc_read_exports(targetd_t)
+rpc_write_exports(targetd_t)
+rpc_manage_exports(targetd_t)
```

Comment:
If I am not mistaken, _manage_ is a complete superset of _write_ here.

Reply:
I actually forgot to amend latest changes which removed the `rpc_write_exports`.  Just refreshed.